src/components/featured-article/macro.spec.js

```diff
@@ -0,0 +1,133 @@
+/** @jest-environment jsdom */
+
+import * as cheerio from 'cheerio';
+import axe from '../../tests/helpers/axe';
+import { renderComponent } from '../../tests/helpers/rendering';
+import { EXAMPLE_FEATURED_ARTICLE_WITH_CHART, EXAMPLE_FEATURED_ARTICLE_WITH_IMAGE } from './_test-examples';
+
+describe('Macro: Featured Article', () => {
+    describe('GIVEN: Params: required', () => {
+        describe('WHEN: chart is provided and image is not', () => {
+            const $ = cheerio.load(renderComponent('featured-article', EXAMPLE_FEATURED_ARTICLE_WITH_CHART));
+
+            test('THEN: it passes accessibility checks', async () => {
+                const results = await axe($.html());
+                expect(results).toHaveNoViolations();
+            });
+
+            test('THEN: it displays the correct title text', () => {
+                expect($('.ons-featured__title').text()).toContain('Economic Trends 2024');
```

Comment:
```suggestion
                expect($('.ons-featured__title').text()).toBe('Economic Trends 2024');
```
Can we not use `toBe` here?